widget/iframe/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+// URL.canParse hasn't good support yet, so using a polyfill if it's not available.
+export function canParse(url: string) {
+  if ('canParse' in window.URL) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
You can delete these comments.

Reply:
It will go through this error:
```
Property 'canParse' does not exist on type '{ new (url: string | URL, base?: string | URL | undefined): URL; prototype: URL; createObjectURL(obj: Blob | MediaSource): string; revokeObjectURL(url: string): void; }'.ts(2339)
```